services.html

```diff
@@ -0,0 +1,89 @@
+        <section class='service-descriptions'>
+          <div class='service-box'>
+            <h3>Digital Design</h3>
+            <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.</p>
+            <a href='#'>Learn More</a>
+          </div>
+          <div class='service-box'>
+            <h3>UX / UI</h3>
+            <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.</p>
+            <a href='#'>Learn More</a>
+          </div>
+          <div class='service-box'>
+            <h3>Digital Marketing</h3>
+            <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.</p>
+          <a href='#'>Learn More</a>
+         </div>
+          <div class='service-box'>
+            <h3>Web Development</h3>
+            <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.</p>
+            <a href='#'>Learn More</a>
+          </div>
+          <div class='service-box'>
+            <h3>iOS Development</h3>
+            <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.</p>
+            <a href='#'>Learn More</a>
+          </div>
+          <div class='service-box'>
+            <h3>Android Development</h3>
+            <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+            eleifend. Mauris euismod facilisis iaculis.</p>
+            <a href='#'>Learn More</a>
+          </div>
+        </section>
+        <div class='line'></div>
```

Comment:
This was a silly oversight from me. I've had a few odd looks about the horizontal line coming from it's own div. It's been low on my priorities since I've been working with this page, but I do plan on getting to it. Thanks for the suggestion.